doc/src/sgml/ref/alter_database.sgml

```diff
@@ -154,6 +154,16 @@ ALTER DATABASE <replaceable class="PARAMETER">name</replaceable> OWNER TO <repla
    User-specific settings override database-specific
    ones if there is a conflict.
   </para>
+
+  <para>
+   Currently RENAME TO is supported only when called by pgdump utility.
+  </para>
+  <para>
+   User can not use "hcatalog" as a name for database, because it's reserved for Hcatalog integration feature.
```

Comment:
The naming convention is supposed to be "HCatalog", not "Hcatalog" - could you please change it in here and the line below?